src/streamlist/StreamList.js

```diff
@@ -34,6 +33,7 @@ export default class StreamList extends React.Component {
             description={showDescriptions && x.description}
             color={x.color}
             isSelected={x.name === selected}
+            isMuted={typeof x.in_home_view !== 'undefined' && !x.in_home_view}
```

Comment:
For some views we don't have that information, and !undefined is true, but we want to not show the mute icon if no data available.